bitmap.go

```diff
@@ -189,11 +189,12 @@ func (b *Bitmap) createSegmentIfNotExists(slice uint64) *BitmapSegment {
 	}
 
 	// Insert new segment.
-	b.segments = append(b.segments, BitmapSegment{})
```

Comment:
change this to NewBTreeBitmap?

Reply:
no segments are containers for bitmaps

Reply:
err, I think I may have put this comment on the wrong line